### PR TITLE
implement String Helper Functions

### PR DESCRIPTION
**Pull request by djy0**
## Pull request type

try to implement #17 

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

**Comment by djy0** [comment]
@jolestar PTAL

File: sources/String.move [hunk 1]
```diff
@@ -0,0 +1,77 @@
+module SFC::String {
+    use StarcoinFramework::Vector;
+    use StarcoinFramework::Debug;
+
+    const HEX_SYMBOLS: vector<u8> = b"0123456789abcdef";
+
+    public fun to_string(value: u128): vector<u8> {
```
Comment (jolestar) [inline sources/String.move:7]: Maybe should return `SFC::ASIIC::String`?

File: sources/String.move [hunk 2]
```diff
@@ -0,0 +1,77 @@
+module SFC::String {
```
Comment (jolestar) [inline sources/String.move:1]: Should we use another module name other than `String`? Maybe in the future, Move has the native string implementation.

Comment (djy0) [reply sources/String.move:1]: maybe use `Strings`, `StringUtil`, `StringHelper` or something else?

File: sources/String.move (hunk 1 above)
Comment (djy0) [reply sources/String.move:7]: > Maybe should return `SFC::ASIIC::String`?

Sure

File: sources/String.move (hunk 2 above)
Comment (jolestar) [reply sources/String.move:1]: StringUtil

**Review by jolestar — APPROVED** [review]

**Pull request merged**